formspecs.lua

```diff
@@ -602,14 +619,19 @@ local get_info_formspec = function(market, account)
 	else
 		formspec[#formspec+1] = "#CCCCCC"..S("No logged activities in this market yet.").."]"
 	end
-	local show_itemnames = account.show_itemnames or "false"
 
+	local show_itemnames = account.show_itemnames or "false"
+	
 	formspec[#formspec+1] = "]container[0.5, 7.6]label[0,0;"..S("Settings")..":]checkbox[0,0.25;show_itemnames;"..S("Show Itemnames")..";"
 		..show_itemnames.."]"
+
+	local show_description = account.show_description or "false"
+	formspec[#formspec+1] = "checkbox[2.1,0.25;show_description;"..S("Show Description")..";"..show_description.."]"
```

Comment:
The other options are in plural, thus "Show Descriptions" would be more consistent.

Reply:
Done